cloudtools/scripts/aws_watch_pending.py

```diff
@@ -155,6 +155,7 @@ def request_spot_instances(all_instances, moz_instance_type, start_count,
     spot_choices = get_spot_choices(connections, spot_rules, product_description)
     if not spot_choices:
         log.warn("No spot choices for %s", moz_instance_type)
+        log.warn("%s - market price too expensive in all available regions", moz_instance_type)
```

Comment:
This is the one I would like to alert on in papertrail. It would be useful to include start_count so we report how many we wanted.